Source/FakeItEasy/Configuration/BuildableCallRule.cs

```diff
@@ -113,7 +113,7 @@ public void WriteDescriptionOfValidCall(IOutputWriter writer)
 
         public virtual void ApplyWherePredicate(Func<IFakeObjectCall, bool> predicate, Action<IOutputWriter> descriptionWriter)
         {
-            this.wherePredicates.Add(Tuple.Create(predicate, descriptionWriter));
+            this.wherePredicates.Add(new Tuple<Func<IFakeObjectCall, bool>, Action<IOutputWriter>>(predicate, descriptionWriter));

```

Comment:
Blow me down. I should've looked for this. We were using it in two out of three places. Should I, as part of a final cleanup commit, make the 3rd (`ZipEnumerable`) use it as well?